src/arrayMethodSort.js

```diff
@@ -4,8 +4,41 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
```suggestion
  [].__proto__.sort2 = function (compareFunction = ...) {
```
let's just add the default value for this function instead of all those checks